src/components/FeatureViews/Dashboard/Dashboard.js

```diff
@@ -37,6 +40,45 @@ const Dashboard = () => {
     const classes = useStyles();
     const user = useSelector(({auth}) => auth) || [];
     const currentDate = moment()
+    let isDisabled;
+
+    const [currentFilter, setCurrentFilter] = useState({
+        showFiltered: false,
+        filter: ""
+    });
+
+    const handleChange = e => {
+        e ? setCurrentFilter({filter: e.label, showFiltered: true}): setCurrentFilter({filter:"", showFiltered: false});
+    };
+
+    const QUERIES = {
+        "categories": gql`query todayCategories {
```

Comment:
Same thing here - you don't need the QUERIES object, if you just have 1 query to call, just create a `const GET_CATEGORIES_OF_TODAY_SESSIONS = gql'query GetTodayCategories{...}'`. Again, be sure to name your variables more descriptively